test.disabled/ejabberd_tests/tests/vcard_SUITE.erl

```diff
@@ -617,12 +617,12 @@ prepare_vcard(ldap, JID, Fields) ->
             undefined ->
                 undefined;
             LdapField ->
-                escalus_ejabberd:rpc(eldap, mod_replace, [LdapField, [Val]])
+                escalus_ejabberd:rpc(eldap, mod_replace, [binary_to_list(LdapField), [binary_to_list(Val)]])
```

Comment:
Would it be better to covert the whole Vcard to list so that there is no need to convert it every time there is a need to use a field?